lookups/ransomware_extensions_lookup.csv

```diff
@@ -300,4 +300,5 @@ Extensions,Name
 *.GANGBANG,Gangbang
 *.reddot,RedDot
 *.MEDUSA,Medusa
-*.rhysida,Rhysida
\ No newline at end of file
+*.rhysida,Rhysida
+*.basta, BlackBasta
```

Comment:
Don't forget to update the corresponding yml file

Reply:
got it. fixed ;)